### ISSUE TITLE
Move the template from the frontpage README to a seperate file

### ISSUE THREAD

**Issue by janhalen** [OS2offdig/governance_report_template#28]
## Rationale
The README in a repo is the welcome page that explains what purpose the repos has and what digital value it delivers.
While it can be argued that it is faster and simpler to access if the README instead is the template it is non-standard and irregular non expected design of a repository. 

## Proposed Solution
The README should be written as a welcoming, and easy to read introduction to the Governance work and have a link to the renamed template near the top and a standard "Getting Started" section that explains how to work with the template.

**Comment by janhalen** [comment]
This is dependending on #27

**Comment by janhalen** [comment]
We decided to name the template file governance_template.md and place it in the root. Make sure that it is easy to find by linking from the top of the Readme ... that is delivered in #27

**Comment by zorp** [comment]
To follow up on this and to understand what is blocking.

@janhalen do I understand correctly that a readme.md file needs to be created and written to complete this issue?

**Comment by janhalen** [comment]
Yes.. we should decide what meets the eye when first hitting this repo. What is the most important to convey about the repo.. I'll create an issue for it.

https://github.com/OS2offdig/governance_report_template/issues/62

It needs to be described

**Comment by zorp** [comment]
Should we complete this issue by renaming the current readme to governance_checklist.md or governance_template.md and then create a new empty readme.md file to be completed in the new issue #62
